### PR TITLE
Replace UILink and a element with MLink

### PR DESCRIPTION
**Pull request by daksas303**
- https://github.com/bratislava/bratislava.sk/issues/1133
- Replace `<a>` and `UILink` with `MLink`

### CODE REVIEW

File: next/components/atoms/CardGradient.tsx
Comment (radoslavzeman) [inline next/components/atoms/CardGradient.tsx:1]: It's obviously not used. You can delete the whole file. :)

File: next/components/ui/RegulationListItem/RegulationListItem.tsx [hunk 1]
```diff
@@ -65,6 +65,7 @@ export const RegulationListItem = ({
         </div>
 
         {mainDocumentHref && (
+          /** Cant find where to test this */
```
Comment (radoslavzeman) [inline next/components/ui/RegulationListItem/RegulationListItem.tsx:68]: This is "VZN" row card, specifically the button on the right.

You can find them here: https://bratislava.sk/mesto-bratislava/sprava-mesta/legislativa-mesta/vseobecne-zavazne-nariadenia

image

Whole VZNs will undergo refactor, so do not use big effort here please :D

**Review by radoslavzeman — COMMENTED** [review]
Please remove also the whole `UIContextProvider` if possible.

File: next/components/atoms/CardGradient.tsx
Comment (daksas303) [reply next/components/atoms/CardGradient.tsx:1]: Deleted

File: next/components/ui/RegulationListItem/RegulationListItem.tsx (hunk 1 above)
Comment (daksas303) [reply next/components/ui/RegulationListItem/RegulationListItem.tsx:69]: Replaced for MLink and fixed TS error.

File: next/components/ui/RegulationListItem/RegulationListItem.tsx [hunk 2]
```diff
@@ -65,10 +66,11 @@ export const RegulationListItem = ({
...
```
Comment (radoslavzeman) [inline next/components/ui/RegulationListItem/RegulationListItem.tsx:69]: Not relevant commnet.

File: next/components/ui/Videos/Videos.tsx [hunk 3]
```diff
@@ -48,11 +49,11 @@ const Video = ({
         allowFullScreen
         onLoad={() => setLoaded(true)}
       />
-      <a href={url ?? undefined} target="_blank" rel="noreferrer">
+      <MLink href={url ?? ''} variant="underlineOnHover" target="_blank" rel="noreferrer">
```
Comment (radoslavzeman) [inline next/components/ui/Videos/Videos.tsx:52]: Please us `'#'` instead of empty string. We had some issues with it.

**Review by radoslavzeman — APPROVED** [review]
Please address comments and you can merge :)

**Pull request merged**